docs/persistentvolumeclaim-metrics.md

```diff
@@ -7,6 +7,7 @@
 | kube_persistentvolumeclaim_labels | Gauge | `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt; <br> `namespace`=&lt;persistentvolumeclaim-namespace&gt; <br> `label_PERSISTENTVOLUMECLAIM_LABEL`=&lt;PERSISTENTVOLUMECLAIM_LABEL&gt;  | STABLE |
 | kube_persistentvolumeclaim_status_phase | Gauge | `namespace`=&lt;persistentvolumeclaim-namespace&gt; <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt; <br> `phase`=&lt;Pending\|Bound\|Lost&gt; | STABLE |
 | kube_persistentvolumeclaim_resource_requests_storage_bytes | Gauge | `namespace`=&lt;persistentvolumeclaim-namespace&gt; <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt; | STABLE |
+| kube_persistentvolumeclaim_status_condition | Gauge | `namespace` =&lt;persistentvolumeclaim-namespace&gt; <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt; <br> `type`=&lt;persistentvolumeclaim-condition-type&gt; <br> `status`=&lt;true\|false\|unknown&gt;  | EXPERIMENTAL |
```

Comment:
In that case kube_persistentvolumeclaim_resource_requests_storage_bytes should also be before kube_persistentvolumeclaim_status_condition and kube_persistentvolumeclaim_status_phase.

Reply:
Done

Reply:
@ankitsinhaiiitg This is not what I had meant actually. You just had to move to `kube_persistentvolumeclaim_status_condition` in between `kube_persistentvolumeclaim_status_phase` and `kube_persistentvolumeclaim_labels` and it would have been in alphabetical order.

Reply:
I agree with you, but the correct alphabetical order would be,
kube_persistentvolumeclaim_access_mode
kube_persistentvolumeclaim_info
kube_persistentvolumeclaim_labels
kube_persistentvolumeclaim_resource_requests_storage_bytes
kube_persistentvolumeclaim_status_condition
kube_persistentvolumeclaim_status_phase
that's what I have changed.